### ISSUE TITLE
Remove/rewrite old tests

### ISSUE THREAD

**Issue by linclark** [firefox-devtools/original-console-fork#219]
This is necessary for the tests to pass when we pref on parts of the new console.

This is being tracked in two ways:
- A [Google doc](https://docs.google.com/spreadsheets/d/1ss3w0jY3S2oZxrdHZioGoZ65EUxYZBMZ_DzEGA6FNtw/edit#gid=0)
- A branch called `tracking-test-cleanup`

When I remove a test, I'll mark it grey in the Google doc. I'll also add a commit to the branch, indicating in the extended commit message why it was removed.

I'll try to comment here when major chunks of work have been done.

**Comment by linclark** [comment]
I've finished the audit of browser mochitests.
- Grey tests have been removed. All of these are either obsolete or should be unit tests and have been captured as future work in this issue queue
- Orange tests likely should remain end-to-end tests. However, I expect them to fail because of `waitForMessages`
- Green tests should likely remain end-to-end tests and I think they should pass.
- Yellow tests are tests for things outside console output. They will be considered later.

My current plan is to move the green tests to a folder in new-console-output tests. If they pass, then I'll move on to trying to do a quick fix for the orange tests and get those committed. Then we can rewrite the grey ones.
